lib/hedwig_sms.ex

```diff
@@ -8,7 +8,7 @@ defmodule Hedwig.Adapters.SMS do
   @doc false
   def init({robot, opts}) do
     HTTPoison.start
-    Hedwig.Robot.register(robot, opts[:name])
+    :global.register_name(opts[:name], robot)
```

Comment:
Looks like `:global.register_name/2` will error if you attempt to register the same process with more than one name.

Looks like it will work with `:global.register_name_external/2`: https://github.com/erlang/otp/blob/maint/lib/kernel/src/global.erl#L312-L343